internal/config/config.go

```diff
@@ -990,6 +991,12 @@ type AmazonRDS struct {
 	RootCertificates string `json:"root_certificates,omitempty" yaml:"root_certificates,omitempty"`
 }
 
+type Service struct {
+	// ApiPrefix prefixes all endpoints (including health and metrics) with its value. It is important to start with `/`.
```

Comment:
💭 Is it really that important that it starts with '/'? We could easily make it start with that if it doesn't

Reply:
Well, we can either enforce it or add `/` ourselves. I personally prefer doing it explicitly than later dig into code to understand why it was added. But this is just my preference. What do you think?